### ISSUE TITLE
PSTREE error

### ISSUE THREAD

**Issue by c0debr** [volatilityfoundation/volatility3#1154]
I am attemtping to use pstree on volatility 2.6.1 and for whatever reason it does not appear to be working despite my using multiple profiles. and i require assistance in trying to find the cause of the error and correcting it?. 
![Screenshot 2024-05-22 182309](https://github.com/volatilityfoundation/volatility3/assets/162753824/851e7e35-0e98-4e41-acf8-f45aa967656e)

**Comment by eve-mem** [comment]
Could you run vol with the -vvv option before the plugin name and share the log output as text and not an image. It really helps work out what's going on.

**Comment by eve-mem** [comment]
A quick note, vol3 doesn't need you to choose a profile. It'll find it automatically.

**Comment by c0debr** [comment]
C:\Users\W0457579\Documents\volatility-master\volatility-master>
C:\Users\W0457579\Documents\volatility-master\volatility-master>
C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win7SP1x64 pstree
Volatility Foundation Volatility Framework 2.6.1
Name                                                  Pid   PPid   Thds   Hnds Time
-------------------------------------------------- ------ ------ ------ ------ ----
 0xfffffa8006d26890:                                    0      0      0 ------ 1970-01-01 00:00:00 UTC+0000

C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win7SP0x64 pstree
Volatility Foundation Volatility Framework 2.6.1
Name                                                  Pid   PPid   Thds   Hnds Time
-------------------------------------------------- ------ ------ ------ ------ ----
 0xfffffa8006d26890:                                    0      0      0 ------ 1970-01-01 00:00:00 UTC+0000

C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win2008R2SP0x64 pstree
Volatility Foundation Volatility Framework 2.6.1
Name                                                  Pid   PPid   Thds   Hnds Time
-------------------------------------------------- ------ ------ ------ ------ ----
 0xfffffa8006d26890:                                    0      0      0 ------ 1970-01-01 00:00:00 UTC+0000

C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win2008R2SP1x64_24000 pstree
Volatility Foundation Volatility Framework 2.6.1
Name                                                  Pid   PPid   Thds   Hnds Time
-------------------------------------------------- ------ ------ ------ ------ ----
 0xfffffa8006d26890:                                    0      0      0 ------ 1970-01-01 00:00:00 UTC+0000

C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win2008R2SP1x64_23418 pstree
Volatility Foundation Volatility Framework 2.6.1
Name                                                  Pid   PPid   Thds   Hnds Time
-------------------------------------------------- ------ ------ ------ ------ ----
 0xfffffa8006d26890:                                    0      0      0 ------ 1970-01-01 00:00:00 UTC+0000

C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win2008R2SP1x64 pstree
Volatility Foundation Volatility Framework 2.6.1
Name                                                  Pid   PPid   Thds   Hnds Time
-------------------------------------------------- ------ ------ ------ ------ ----
 0xfffffa8006d26890:                                    0      0      0 ------ 1970-01-01 00:00:00 UTC+0000

C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win7SP1x64_24000 pstree
Volatility Foundation Volatility Framework 2.6.1
Name                                                  Pid   PPid   Thds   Hnds Time
-------------------------------------------------- ------ ------ ------ ------ ----
 0xfffffa8006d26890:                                    0      0      0 ------ 1970-01-01 00:00:00 UTC+0000

C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win7SP1x64_23418 pstree
Volatility Foundation Volatility Framework 2.6.1
Name                                                  Pid   PPid   Thds   Hnds Time
-------------------------------------------------- ------ ------ ------ ------ ----
 0xfffffa8006d26890:                                    0      0      0 ------ 1970-01-01 00:00:00 UTC+0000

C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win7SP1x64 pslist
Volatility Foundation Volatility Framework 2.6.1
Offset(V)          Name                    PID   PPID   Thds     Hnds   Sess  Wow64 Start                          Exit
------------------ -------------------- ------ ------ ------ -------- ------ ------ ------------------------------ ------------------------------
0xfffffa8006d26890                           0      0      0 -------- ------      0

C:\Users\W0457579\Documents\volatility-master\volatility-master>

does this work better?

**Comment by c0debr** [comment]
and i understand but i simply wish to work out whats wrong with 2.6

**Comment by eve-mem** [comment]
Please use the -vvv option so that more logs are made and upload that. 

Vol3 has a 2.6.1 version, or do you mean you're trying to run the old python2 framework? If so you'll have more luck raising an issue there.

**Comment by c0debr** [comment]
how do i use the -vvv option?

**Comment by c0debr** [comment]
C:\Users\W0457579\Documents\volatility-master\volatility-master>python vol.py -f "Clone of Clone of Windows 7 x64-a88debbd.vmem" --profile=Win7SP1x64 -vvv pslist
Volatility Foundation Volatility Framework 2.6.1
Offset(V)          Name                    PID   PPID   Thds     Hnds   Sess  Wow64 Start                          Exit
------------------ -------------------- ------ ------ ------ -------- ------ ------ ------------------------------ ------------------------------
0xfffffa8006d26890                           0      0      0 -------- ------      0

C:\Users\W0457579\Documents\volatility-master\volatility-master>

alright here's what i was able to find

**Comment by c0debr** [comment]
also where do you reccomend i ask this issue?

**Comment by eve-mem** [comment]
Yes, that is the older Framework. I can't personally help you with that. You'll have more luck using the newer framework or raising an issue on the older frameworks page. 

This isn't an issue with vol3.

**Comment by c0debr** [comment]
where's the older frameworks page?

**Comment by eve-mem** [comment]
https://github.com/volatilityfoundation/volatility

**Comment by c0debr** [comment]
how do i comment on the page since im not seeing the option to do so?

**Comment by eve-mem** [comment]
In exactly the same way you created this issue, just on that project instead https://github.com/volatilityfoundation/volatility/issues

I'd really recommend using the newer framework.
